apps/modernization-ui/src/apps/patient/add/extended/inputs/address/AddressRepeatingBlock.tsx

```diff
@@ -5,18 +5,19 @@ import { AddressView } from './AddressView';
 import { AddressEntryFields } from 'apps/patient/data/address/AddressEntryFields';
 import { RepeatingBlock } from 'design-system/entry/multi-value/RepeatingBlock';
 import { ReactNode } from 'react';
+import { asSelectable } from 'options';
 
 const defaultValue: Partial<AddressEntry> = {
     asOf: today(),
-    type: undefined,
-    use: undefined,
+    type: asSelectable(''),
+    use: asSelectable(''),
```

Comment:
issue: This cannot be an empty `Selectable`, it has to be `null` or `undefined`.  An empty `Selectable` is a value which is bypassing the validations.

![image](https://github.com/user-attachments/assets/6f0e565d-f7f7-49bb-b32c-c91c1053099f)